fetchai/registration.py

```diff
@@ -140,3 +164,4 @@ def register_with_agentverse(
         },
     )
     r.raise_for_status()
+    logger.info("Completed registering agent with Agentverse")
```

Comment:
d'you want to stick `registration_metadata` on this one for context as well?

Reply:
[90701d7](https://github.com/flockx-official/fetchai/pull/29/commits/90701d7a3f22b08ea0255b54fd21c1efa844b143)